README.md

```diff
@@ -34,6 +34,30 @@ To run PlaceCal locally you will need:
 
 With that said, here's what you need to get rolling.
 
+### Set up postgresql server (Via docker)
+
+**Note: Skip this step if you're using a system-installed docker**
+
+Creating a postgres docker image is reasonably quick:
+
+``` sh
+docker network create placecal-network
+docker create --name placecal-db --network placecal-network --network-alias postgres -p 5432:5432 --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -e 'POSTGRES_DB=placecal_db' -e 'POSTGRES_USER=postgres' -e 'POSTGRES_PASSWORD=foobar' -e 'POSTGRES_PORT=5432' postgres:14.1
+docker start placecal-db
+```
+
+You should now set the following values in your `.env`:
```

Comment:
Should this be inside the project folder, ie after the git clone? could you just rename the `.env.example` file?

Reply:
have updated this - open a new comment if unclear